test/python/test_instructions.py

```diff
@@ -386,6 +391,12 @@ def test_sign_attestation(
         snap_path=snap_path / "app_context"
     )
 
+    tezos_navigator.backend.right_click()
+    tezos_navigator.assert_screen(
+        name="home_screen",
+        snap_path=snap_path / "app_context"
+    )
+ 
```

Comment:
Since this code is duplicated, I think it could be worth to add a `black_screen` boolean argument (default false) to `check_app_context` and move the :
```python
    tezos_navigator.assert_screen(
        name="black_screen",
        snap_path=snap_path / "app_context"
    )

    tezos_navigator.backend.right_click()
    
    tezos_navigator.assert_screen(
        name="home_screen",
        snap_path=snap_path / "app_context"
    )
```
at the start of the home screen checks in `check_app_context`

Reply:
Done.